lib/minitest.rb

```diff
@@ -507,9 +510,21 @@ def record result # :nodoc:
   #
   # If you want to create an entirely different type of output (eg,
   # CI, HTML, etc), this is the place to start.
+  #
+  # Example:
+  #
+  #   class JenkinsCIReporter < StatisticsReporter
+  #     def report
+  #       super  # Needed to calculate some statistics
+  #
+  #       print "<testsuite "
+  #       print "tests='#{count}' "
+  #       print "failures='#{failures}' "
+  #       # Remaining XML...
+  #     end
+  #   end
 
   class StatisticsReporter < Reporter
-    # :stopdoc:
```

Comment:
This drops `rake dcov` from 100% to 96.24%. ☹️

Reply:
I can add docs to the `attr_accessor`s, but they're already self explanatory, don't you think?

Reply:
no. I don't think. As you can see in the rest of the library.

Reply:
Then as someone that does (from time to time) I would suggest ignoring the coverage here.